anonymizer/base.py

```diff
@@ -71,9 +71,11 @@ def get_allowed_value(self, source, field):
 
     ### Public interace ##
     def uuid(self, field=None):
-        def source():
-            return str(uuid4())
-        return self.get_allowed_value(source, field)
+        retval = str(uuid4())
+        max_length = getattr(field, 'max_length', None)
+        if max_length is not None:
+            retval = retval[:max_length]
```

Comment:
uuid1 is awful if you want something close to a "random distribution". Look at this.

![image](https://cloud.githubusercontent.com/assets/379257/11882045/22cf5716-a4bf-11e5-88ca-e07c57368cf2.png)